core/db/migrate/20210722090705_add_store_id_to_spree_customer_returns.rb

```diff
@@ -0,0 +1,11 @@
+class AddStoreIdToSpreeCustomerReturns < ActiveRecord::Migration[6.1]
+  def change
+    unless column_exists?(:spree_customer_returns, :store_id)
+      add_column :spree_customer_returns, :store_id, :bigint
+      add_index :spree_customer_returns, :store_id
+      add_index :spree_customer_returns, [:number, :store_id], unique: true
+      Spree::CustomerReturn.reset_column_information
+      Spree::Taxonomy.update_all(store_id: Spree::Store.default.id)
```

Comment:
@szymoniwacz we need to find the proper store via order associated to customer return